README.md

```diff
@@ -230,6 +234,29 @@ chmod +x summon-cli-linux
 mv summon-cli-linux ~/.local/bin/summon
 ```
 
+##### TUI: Homebrew [↑](#structure)
+
+If you are on MacOS, you can get Summoner using Homebrew Kowainik's Tap.
+
+You need to run the following commands for that:
+
+```shell
+$ brew tap kowainik/tap
+$ brew install summoner-tui
```

Comment:
For future reference these two commands can be reduced to simply:

```sh
brew install kowainik/tap/summoner-tui
```

😄

Reply:
Oh, nice! 👍